content/en/blog/_posts/2023-11-xx-scheduler-queueinghint/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog
+title: "Kubernetes v1.28: QueueingHint brings a new possibility to optimize our scheduling"
+date: 2023-10-25T10:00:00-08:00
+slug: scheduler-queueinghint
+---
+
+**Author:** [Kensei Nakada](https://github.com/sanposhiho) (Mercari)
+
+The scheduler is the core component that decides which Node Pods run on.
+Basically, it schedules Pods **one by one**, 
+and thus the larger your cluster is, the more crucial the throughput of the scheduler is.
+
+The throughput of the scheduler is our eternal challenge, 
+over the years, SIG-Scheduling have been putting effort to improve the scheduling throughput by many enhancements.
+
+In this blog post, I'll introduce a recent major improvement in the scheduler, named QueueingHint.
+
+We'll go through the explanation of the basic background knowledge of the scheduler,
+and how QueueingHint improves our scheduling throughput.
+
+## Scheduling Queue
+
+The scheduler has Scheduling Queue which has all unscheduled Pods.
+
+Scheduling Queue is composed of three places in it - ActiveQ, BackoffQ and Unschedulable Pod Pool.
+- ActiveQ: Pods which are ready to get scheduling.
+- BackoffQ: Pods which are waiting for the backoff, and will be put into ActiveQ after that.
+- Unschedulable Pod Pool: Pods which should not be scheduled for now.
```

Comment:
```suggestion
- Unschedulable Pod Pool: It holds Pods which should not be scheduled for now, because they have a Scheduling Gate or because the scheduler attempted to schedule them and nothing has changed in the cluster that could make the Pod schedulable.
```